docs/pages/access-controls/access-request-plugins/ssh-approval-mattermost.mdx

```diff
@@ -174,12 +201,35 @@ files and leave `identity` commented out.
 `identity` field with the path to the identity file you exported earlier and
 comment out the other fields.
 
+</TabItem>
+<TabItem label="Teleport Enterprise " scope={["oss", "enterprise"]} options="Helm Chart">
+
+**`address`**: Include the hostname and HTTPS port of your Teleport Cloud tenant
+(e.g., `mytenant.teleport.sh:443`).
+
+**`identitySecretName`**: Fill in the `identitySecretName` field with the name
+of the Kubernetes secret you created earlier.
+If you exported an identity file earlier, fill in the `identity` field with the
+path to the file and comment out the other fields.
+
+</TabItem>
+<TabItem label="Teleport Cloud " scope="cloud" options="Helm Chart">
+
+**`address`**: Include the hostname and HTTPS port of your Teleport Cloud tenant
+(e.g., `mytenant.teleport.sh:443`).
+
+**`identitySecretName`**: Fill in the `identitySecretName` field with the name
+of the Kubernetes secret you created earlier.
+
 </TabItem>
 </Tabs>
 
 
 ### `[mattermost]`
 
+<Tabs dropdownCaption="Environment type" dropdownSelected="Executable">
```

Comment:
I think we can use a regular `Tabs` component here, without a dropdown menu, since there is one tab for a Helm chart and one tab for an executable

Reply:
Yeah, I was thinking about keeping the different axes on their "original" place, creating some consistency throughout the document, but let's try the other way. I'll change it.